Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -2857,7 +2857,7 @@ module.exports = {
    *```javascript
    *  onScroll={Animated.event(
    *    [{nativeEvent: {contentOffset: {x: this._scrollX}}}],
-   *    {listener},          // Optional async listener
+   *    {listener: (event, gestureState) => console.log(gestureState)}, // Optional async listener
```

Comment:
Since this looks more an onScroll event from ScrollView I think we should not add the gestureState param, might be more confusing than anything else. Maybe just `(event) => console.log(event)`